ipatests/test_integration/test_acme.py

```diff
@@ -122,21 +122,23 @@ def certbot_register(host, acme_server):
     )
 
 
-def certbot_standalone_cert(host, acme_server):
+def certbot_standalone_cert(host, acme_server, no_of_cert=1):
     """method to issue a certbot's certonly standalone cert"""
     # Get a cert from ACME service using HTTP challenge and Certbot's
     # standalone HTTP server mode
     host.run_command(['systemctl', 'stop', 'httpd'])
-    host.run_command(
-        [
-            'certbot',
-            '--server', acme_server,
-            'certonly',
-            '--domain', host.hostname,
-            '--standalone',
-            '--key-type', 'rsa',
-        ]
-    )
+    for i in range(0, no_of_cert):
+        host.run_command(
+            [
+                'certbot',
+                '--server', acme_server,
+                'certonly',
+                '--domain', host.hostname,
+                '--standalone',
+                '--key-type', 'rsa',
+            ],
+            stdin_text='2'
```

Comment:
The `certonly` sub command accepts `--force-renewal` if you want to avoid using stdin_text.